config/routes.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+end
```

Comment:
- [ Required ] Good job making some of the changes from the previous reviewer, however, I noticed that your application has no routes in `routes.rb`. For the devise gem to work correctly you have to set up the `sign_up` and `sign_in` routes which help users to `register` and `login` in to your application. I would like to suggest that you follow any of these articles step by step [Article One >>](https://hackernoon.com/using-devise-in-your-ruby-on-rails-application-a-step-by-step-guide-m92i3y5s) or [Article Two](https://github.com/heartcombo/devise) to ensure that you setup  devise correctly. 

- Please ensure that the `login` and `register` pages are styled according to the wireframes provided for this project as  requested by the previous reviewer, I could not review the design of the `login` or `register` pages because there are no routes in your application that lead to these pages.

## Error when trying to register on the App
![Screenshot from 2023-12-15 01-59-25](https://github.com/obibaadoma/Blog_App/assets/16392046/c7af4635-f27c-48c6-b1f1-81688fabf4f3)

## Error when trying to login on the App
![Screenshot from 2023-12-15 02-02-36](https://github.com/obibaadoma/Blog_App/assets/16392046/07c4ad85-c107-4da8-b60d-536d16e0fcab)